### DISCUSSION TITLE
version of the app in UI and linux mint software manager

### DISCUSSION THREAD

**Discussion by YukiAlmasy** [jpochyla/psst#310] [General]
could we put the version of the app you have installed in the actual UI of the app. please? also, for some reason when I installed this on linux mint 20.3, the software manager identifies it as a completely different program that is also known as psst, weird.

**Comment by jpochyla** [comment]
Hi @YukiAlmasy, I'm sorry, but Psst does not havy any releases now. There is a [Beta milestone](https://github.com/jpochyla/psst/milestone/1) for this.
